pkg/hosts/hosts.go

```diff
@@ -91,3 +100,14 @@ func (h *Hosts) Clean(rawSuffixes []string) error {
 	}
 	return h.File.Flush()
 }
+
+func (h *Hosts) checkIsWritable() error {
+	if !h.File.IsWritable() {
+		return fmt.Errorf("host file not writable, try running with elevated privileges")
+	}
+	return nil
+}
+
+func (h *Hosts) Contains(ip, host string) bool {
+	return h.File.Has(ip, host)
```

Comment:
Initially i thought maybe just checking only the hostname exists would be enough..